docs/understanding-drips/faq.md

```diff
@@ -2,25 +2,11 @@
 id: faq
 title: FAQ
 ---
-## I'm a creator/developer. How do I raise funds using Drips?
-Head over to the [Create a Community][cc] section for instructions on how to
-set up your Community and issue member tokens for your fans to purchase.
-
-## I want to purchase a membership to a Community I'm a fan of. How do I do that?
-Please see the [Purchase a Membership][pm] section for instructions on how to
-purchase a membership token for a Community you wish to support.
-
-## Is it posssible to drip funds using ETH or an ERC-20 token other than DAI?
-In the current version of Drips, DAI is the only token that can be dripped to other
-users or Communities. However the Drips team is actively working to add support for
-dripping other types of tokens in the next version of the protocol. 
-
-## Do I need to have Eth in my account to send or receive DAI using Drips?
-Users of Drips do need to have Eth (or MATIC for the Polygon version of Drips)
-available in their wallet to cover transaction fees associated with creating Drips (if sending
-DAI) or collecting (if receiving DAI). 
-
-
-
-[cc]: using-drips/create-a-community.md
-[pm]: using-drips/purchase-a-membership.md
\ No newline at end of file
+### Are there fees for using Drips?
+The Drips Protocol is free to use and does not impose any fees on users. However, users will still need to pay gas
+fees to interact with the Drips Contracts as they would with any contract on Ethereum. Also, users should be aware that third-party apps building on
+Drips may choose to impose fees of their own.
+
+### Do I need to have Eth in my account to send or receive DAI using Drips?
```

Comment:
Should we restruct this for any ERC20 instead of DAI?

Reply:
Fixed this